File: library/src/main/java/me/proxer/library/api/media/InternalApi.java
package me.proxer.library.api.media;

import me.proxer.library.api.ProxerCall;
import me.proxer.library.entity.media.CalendarEntry;
import retrofit2.http.GET;

import javax.annotation.ParametersAreNullableByDefault;
import java.util.List;

/**
 * @author Ruben Gees
 */
@ParametersAreNullableByDefault
public interface InternalApi {

    @GET("media/calendar")
    ProxerCall<List<CalendarEntry>> calendar();
}

File: library/src/main/kotlin/me/proxer/library/api/media/InternalApi.kt
package me.proxer.library.api.media

import me.proxer.library.ProxerCall
import me.proxer.library.entity.media.CalendarEntry
import retrofit2.http.GET

/**
 * @author Ruben Gees
 */
internal interface InternalApi {

    @GET("media/calendar")
    fun calendar(): ProxerCall<List<CalendarEntry>>
}
